specification/monitor/resource-manager/Microsoft.Monitor/preview/2021-06-03-preview/monitoringAccounts_API.json

```diff
@@ -0,0 +1,556 @@
+        "responses": {
+          "200": {
+            "description": "Monitoring account was successfully updated",
+            "schema": {
+              "$ref": "#/definitions/MonitoringAccountResource"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Update monitoring account": {
+            "$ref": "./examples/MonitoringAccountsUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "MonitoringAccounts"
+        ],
+        "description": "Deletes an monitoring account",
+        "operationId": "MonitoringAccounts_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/MonitoringAccountNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Monitoring account was successfully deleted"
+          },
+          "204": {
+            "description": "Monitoring account did not exist"
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete monitoring account": {
+            "$ref": "./examples/MonitoringAccountsDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Metrics": {
+      "description": "Information about metrics for the monitoring account",
+      "type": "object",
+      "properties": {
+        "prometheusQueryEndpoint": {
+          "description": "The Prometheus query endpoint for the monitoring account",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        },
+        "internalId": {
+          "description": "An internal identifier for the metrics container. Only to be used by the system",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-mutability": [
+            "read"
+          ]
+        }
+      }
+    },
+    "IngestionSettings": {
+      "description": "Settings for data ingestion",
+      "type": "object",
+      "properties": {
+        "dataCollectionRule": {
```

Comment:
>dataCollectionRule [](http://example.com/codeflow?start=9&length=18)

consider endoing this name with "...resourceId"

Reply:
Renamed this property